cmd/cloud-api-adaptor/main.go

```diff
@@ -57,7 +57,8 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 			flags.StringVar(&hypcfg.HypProvider, "provider", "aws", "Hypervisor provider")
 			flags.StringVar(&cfg.TunnelType, "tunnel-type", podnetwork.DefaultTunnelType, "Tunnel provider")
 			flags.StringVar(&cfg.HostInterface, "host-interface", "", "Host Interface")
-
+			// TODO: REMOVE ME AFTER cc supported pass image name rather than digest for existing image
+			flags.StringVar(&hypcfg.CriSocketPath, "runtime-endpoint", "/var/run/crio/crio.sock", "cri runtime uds endpoint")
```

Comment:
Cloud you use the same default value for all three hypervisor types?

Reply:
@yoheiueda I was thinking aws and libvirt are using cri-o as CRI runtime, is that the case?

Reply:
Sorry, let me change my comment.

Can we change this feature opt-in as I said previously? That means we don't set any default value for this option.

Reply:
Looks, aws is using cri-o and ibmcloud is using containerd, while libvirt is mostly used for debug/develop. How about we leave as empty for libvirt? @yoheiueda I made the change in new commits.

Reply:
My point is that this feature is a kind of hacks and has a complicated data flow as you pointed out. Users may use cri-o on IBM Cloud, and also may use containerd on AWS.

Instead of automatically fixing the issue, I think we should let users explicitly enable this feature when they need to fix the issue. What do you think?

Reply:
Got your point, yeah, lets leave it empty and as an optional "opt-in" parameters as this is a workaround.